src/client/renderer/r_entity.c

```diff
@@ -26,25 +26,7 @@
  */
 static void R_SetEntityBounds(r_entity_t *e) {
 
-	e->abs_model_mins = Vec3_Mins();
-	e->abs_model_maxs = Vec3_Maxs();
-
-	const vec3_t corners[] = {
-		Vec3(e->model->mins.x, e->model->mins.y, e->model->mins.z),
-		Vec3(e->model->maxs.x, e->model->mins.y, e->model->mins.z),
-		Vec3(e->model->maxs.x, e->model->maxs.y, e->model->mins.z),
-		Vec3(e->model->mins.x, e->model->maxs.y, e->model->mins.z),
-		Vec3(e->model->mins.x, e->model->mins.y, e->model->maxs.z),
-		Vec3(e->model->maxs.x, e->model->mins.y, e->model->maxs.z),
-		Vec3(e->model->maxs.x, e->model->maxs.y, e->model->maxs.z),
-		Vec3(e->model->mins.x, e->model->maxs.y, e->model->maxs.z),
-	};
-
-	for (size_t i = 0; i < lengthof(corners); i++) {
-		const vec3_t corner = Mat4_Transform(e->matrix, corners[i]);
-		e->abs_model_mins = Vec3_Minf(e->abs_model_mins, corner);
-		e->abs_model_maxs = Vec3_Maxf(e->abs_model_maxs, corner);
-	}
+	e->abs_model_bounds = Mat4_TransformBounds(e->matrix, e->model->bounds);
```

Comment:
Are you sure this correctly handles rotations? The above code took me quite some time to sort out. The code isn't rocket science, but due to rotations, it's not as easy as "subtract mins and add maxs from / to origin." Getting this wrong will mean tons of regressions around culling, occluding, etc.